changelogs/unreleased/global-names.md

```diff
@@ -0,0 +1,20 @@
+## feature/replication
+
+* New option `box.cfg.cluster_name` allows to assign a human-readable name to
+  the entire cluster with all its replicasets. It has to match in all instances
+  of all the replicasets in one cluster and is displayed in
+  `box.info.cluster.name` (gh-5029).
```

Comment:
```suggestion
* A new option `box.cfg.cluster_name` allows assigning a human-readable name
  to the entire cluster. Its value must be the same across all instances in
  the cluster. The cluster name is displayed in `box.info.cluster.name` (gh-5029).
```

Reply:
Phrases "the entire cluster with all its replicasets" and "all instances of all the replicasets in one cluster" contain repetitions of the same thought, so I suggest shortening this text.